cmd/kops/manifest_test.go

```diff
@@ -0,0 +1,84 @@
+package main
+
+// Manifest CRUD tests
+// Note: The CRUD operations are temporarily skipped until Kops can support a testing environment
+// to enable these tests remove the `t.SkipNow()` methods, and flesh out manifest_example.yml with
+// a valid test environment configuration.
+// - or -
+// Create a new manifest testing .yml file and update the manifest var
+
+import (
+	"testing"
+)
+
+var manifest = "../../manifest_example.yml"
```

Comment:
Should we allow this to be an ENV VAR?  That way we can dynamically set our test files?